modules/logging-output-types.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/logging-multi-clf.adoc
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_content-type: REFERENCE
+[id="logging-output-types_{context}"]
+= Log output types
+
+The log output specified in the `ClusterLogForwarder` CR can be any of the following types:
+
+`default`:: The on-cluster, Red{nbsp}Red managed log store. You are not required to configure the default output.
++
+[NOTE]
+====
+If you try to configure a `default` output, you receive an error message, because the `default` output is reserved for the Red{nbsp}Hat OpenShift Logging Operator.
+====
+`loki`:: Loki, a horizontally scalable, highly available, multi-tenant log aggregation system.
```

Comment:
There are a few missing from this list https://github.com/openshift/cluster-logging-operator/blob/master/docs/features/collection.adoc#outputs but please ignore if this is just a "move"

Reply:
This is just a move, but maybe we can update those as part of https://issues.redhat.com/browse/OBSDOCS-104? Added a comment to that issue to include info from this link.